Extension/package.json

```diff
@@ -457,6 +457,17 @@
             ],
             "scope": "resource"
           },
+          "C_Cpp.markdownCommentFormatting": {
```

Comment:
Formatting in the settings currently refers to actual formatting of code, so it seems like it be less confusing this were named something like "markdownCommentRendering" or "displayMarkdownComments" or "renderMarkdownComments", etc.